modules/about-must-gather.adoc

```diff
@@ -57,3 +57,11 @@ openshift-must-gather-5drcj    must-gather-s8sdh    2/2     Running     0
 ...
 ----
 // todo: table or ref module listing available images?
+Optionally, you can run the `oc adm must-gather` command in a specific namespace by using the `--run-namespace` option.
```

Comment:
```suggestion
Optionally, you can use the `--run-namespace` option to specify a namespace to run the `must-gather` tool in.
```

Reply:
Thanks!

Reply:
Are you going to implement Andrea's feedback?

Reply:
We talked about it and she said it was fine the way it was. Would you like me to change it?

Reply:
Yep we talked about it and I think this is good the way it is. Thanks for checking!